x-pack/test/functional_with_es_ssl/apps/ml/alert_flyout.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+  const ml = getService('ml');
+  const pageObjects = getPageObjects(['triggersActionsUI']);
+
+  describe('anomaly detection alert', () => {
```

Comment:
Since there's no top level suite in `functional_with_es_ssl`, each app has to define the `ciGroup` individually. In the current state I think the ML alerting suite is not executed in CI.
Looking at recent ciGroup execution times and taking into account that ML UI tests are currently executed in xpack groups 3 and 13, I'd suggest to add
```
this.tags('ciGroup13');
```
here.

Reply:
Added in https://github.com/elastic/kibana/pull/89286/commits/3e569c79418823882d7892129037866ac5c1bf0c